core/src/main/java/org/apache/iceberg/rest/RESTMessage.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.rest;
+
+/**
+ * Interface to mark both REST requests and responses.
+ */
+public interface RESTMessage {
+}
```

Comment:
If we want to simplify, we can remove this and then have `RESTRequest` and `RESTResponse` be entirely independent interfaces for the moment.

But I think they should be tied together, as they share the same structure overall.

Reply:
I'd probably add the `validate` method now. Otherwise, I'm not sure what the value of this interface is.

Reply:
Well, the interface would still have value for type restraints even if it were empty.

But I agree that `validate` is good to include now.